File: lab1.cpp
#include <iostream>
using namespace std;

// this is class
class Triangle{
public:

float width,height;

//this is constructor
Triangle(float w,float h){
width=w;
height=h;
}

//this is method
float calculateArea() {
return 0.5*width*height;
}
};

//this is main function
int main() {
float width,height;
  
cout << ("Enter the value of a width=");
cin >> width;
cout << ("Enter the value of a height=");
cin >> height ;

//this is object
Triangle tri (width,height);
cout << "Area of the Triangle" << tri.calculateArea() << endl;
return 0;
}

File: Lab1.java
1. Write a program to print the area of a triangle having width and height by creating a class
name ‘Triangle’ with parameter in its constructor. ( By C++ or Java )



import java.util.Scanner;  //package

//This is class
public class Triangle{
    double width,height;

  //this is constructor
   public Triangle(double width,double height){
        this.width=width;
        this.height=height;
    }

    //this is method
    public double calculateArea(){
        return 0.5*width*height;
    }

  //this is main function
    public static void main(String[] args){
       double width,height;
       
       Scanner scanner=new Scanner(System.in);
       
       System.out.println("Enter the value of width =");
       width=scanner.nextDouble();
       
       System.out.println("enter the value of height =");
       height=scanner.nextDouble();

      //this is object
       Triangle triangle=new  Triangle (width,height);

       System.out.println("Area of the triangle" + triangle.calculateArea());
       
       scanner.close();
       
    }
}
